templates/nginx.conf.j2

```diff
@@ -100,7 +96,7 @@ server {
 
 {% if cert_source != 'none' %}
 server {
-    listen 443;
+    listen {{ nginx_https_port }} ssl;
```

Comment:
What does the extra ` ssl` at the end of this line signify? (Mentioning it because I don't see it in the existing config, so want to be sure it doesn't break existing installations)

Reply:
The `ssl` on the end of the `listen` says that connections on this port require SSL.  It's superfluous here since the whole server section has `ssl on` (lower down) meaning that all connections for the server require SSL, but it's the newer, more flexible way of doing things that allows SSL on some ports and not others within the same server.  I probably copied it from another configuration or something like that, so I'll remove it here to avoid confusion.  It's not necessary for this PR.